### ISSUE TITLE
Binance top of the book (best ask/bid) access.

### ISSUE THREAD

**Issue by elabunsky** [bmoscon/cryptofeed#685]
Hello,

Could you advice me how I access Binance top of the book quotes instead of L2? Top of the book updates real time and L2 each 100ms only.

Regards,
Eugene.

**Comment by elabunsky** [comment]
Will try. Thank you for advice.

**Comment by bmoscon** [comment]
that version is so old (over a year ago, that was may of 2020) its probably not going to support it. Try one of the later releases.

**Comment by bmoscon** [comment]
I've confirmed this works in later releases. As far as "real time" - can't get the data any faster than binance provides it. They limit their book updates to 100ms, if they similarly limit their ticker data, not much I can do about it. Going to close this, but if you have more questions ask them here or on slack and I'll help out if I can

**Comment by elabunsky** [comment]
> I've confirmed this works in later releases. As far as "real time" - can't get the data any faster than binance provides it. They limit their book updates to 100ms, if they similarly limit their ticker data, not much I can do about it. Going to close this, but if you have more questions ask them here or on slack and I'll help out if I can

@bmoscon you are not right - even you will subscribe top of the book (best ask/bid) or All Book Tickers Stream
 binance and binance_futures will send it realtime and if you will subscribe Partial Book Depth Streams (Top bids and asks, Valid are 5, 10, or 20) they will send updates each 100 ms. 

PS. How I can install last CryptoFeed? I used pip3 3 weeks ago and installed 1 year old package ;(

Regards,
Eugene.
